Library/Formula/mono.rb

```diff
@@ -30,7 +33,13 @@ def install
       --prefix=#{prefix}
       --enable-nls=no
     ]
-    args << "--build=" + (MacOS.prefer_64_bit? ? "x86_64": "i686") + "-apple-darwin"
+
+    if build.with? '64bit'
+      args << "--build=x86_64-apple-darwin"
+    else
+      # no option specified
+      args << "--build=" + (MacOS.prefer_64_bit? ? "x86_64": "i686") + "-apple-darwin"
```

Comment:
I think it should default to whatever the brew infrastructure has as defaults, like it is here, because otherwise, the user might as well just unlink the formula and use `brew cask install xamarin-studio` with its 32 bits installation -- also that installation does a lot more than just install the mono framework, so it doesn't make sense to adapt the mono framework installation after the studio, unless it's the studio that's being installed.